curriculum/Chapter-2-Agile-Team-Collaboration.md

```diff
@@ -0,0 +1,140 @@
+# Chapter 2: Agile and Team Collaboration
+
+## Objectives
+
+By completing this chapter, you will:
+
+- Understand the foundational Agile Methodologies and the specific Rise8 Agile Principles.
+- Explore the dynamics of Agile and Team Collaboration.
+- Recognize the responsibilities within Agile teams, including partner engagement and portfolio leadership.
```

Comment:
We should cut this.

Reply:
Refactor:

Explore the dynamics of agile and team responsibilities and collaboration.

Reply:
Agile*